src/core_atmosphere/physics/mpas_atmphys_camrad_init.F

```diff
@@ -789,30 +779,40 @@ subroutine oznini(mesh,atm_input)
  call mpas_pool_get_array(atm_input,'ozmixm',ozmixm)
 
 !-- read in ozone pressure data:
- open(pin_unit,file='OZONE_PLEV.TBL',action='READ',status='OLD',iostat=istat)
+ if(dminfo % my_proc_id == IO_NODE) then
+    call mpas_new_unit(read_unit)
+    if(read_unit < 0) &
+       call physics_error_fatal('module_ra_cam: oznini: Cannot find unused '//&
+                                'fortran unit to read in lookup table.')
+ end if
+
+!distribute unit to other processors:
+ call mpas_dmpar_bcast_int(dminfo,read_unit)
```

Comment:
I don't think this is guaranteed to be correct. It's possible that on MPI rank 0, unit *N* is available, but that unit *N* is not available on one or more other ranks. In other words, we cannot assume that the usage of I/O units is identical across all MPI ranks.

Regardless, I think we should stick to the existing pattern of having just MPI rank 0 read data and broadcast that data to all other ranks.